enclave/sgx/attester.c

```diff
@@ -345,30 +378,49 @@ static oe_result_t _get_attester_plugins(
     uuid_list = (oe_uuid_t*)temporary_buffer;
     uuid_count = temporary_buffer_size / sizeof(oe_uuid_t);
 
-    // Add one additional entry: the first one for local attestation
-    *attesters =
-        (oe_attester_t*)oe_malloc(sizeof(oe_attester_t) * (uuid_count + 1));
+    // For SGX ECDSA_p256, add support of OE report and SGX quote
+    for (size_t i = 0; i < uuid_count; i++)
+        if (!memcmp(uuid_list + i, &_ecdsa_uuid, sizeof(oe_uuid_t)))
+        {
+            legacy_uuid_count = 2;
+            break;
+        }
```

Comment:
Sorry I just got confused here. So if _ecdsa_uuid is included, we add two more slots for OE report and SGX quote plugins. If not, we don't. In which case _ecdsa_uuid might not be included in sgx attester? Can't we just set `legacy_uuid_count = 2`?

Reply:
If a platform has neither SGX DCAP nor quote-ex libraries installed, it would not have support of _ecdsa_uuid. Adding comments to clarify.

Reply:
Got it. Thanks!